cmd/api/main.go

```diff
@@ -45,17 +42,20 @@ import (
 	"google.golang.org/grpc/reflection"
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
 )
 
 type ConfigFile struct {
-	DB_USER     string `mapstructure:"DB_USER"`
-	DB_PASSWORD string `mapstructure:"DB_PASSWORD"`
-	DB_PROTOCOL string `mapstructure:"DB_PROTOCOL"`
-	DB_ADDR     string `mapstructure:"DB_ADDR"`
-	DB_NAME     string `mapstructure:"DB_NAME"`
-	DB_SSLMODE  string `mapstructure:"DB_SSLMODE"`
+	DB_USER         string `mapstructure:"DB_USER"`
+	DB_PASSWORD     string `mapstructure:"DB_PASSWORD"`
+	DB_HOST         string `mapstructure:"DB_HOST"`
+	DB_PORT         string `mapstructure:"DB_PORT"`
+	DB_NAME         string `mapstructure:"DB_NAME"`
+	DB_SSLMODE      string `mapstructure:"DB_SSLMODE"`
+	GRPC_PORT       string `mapstructure:"GRPC_PORT"`
+	REST_PORT       string `mapstructure:"REST_PORT"`
+	PROMETHEUS_PORT string `mapstructure:"PROMETHEUS_PORT"`
+	TLS_OVERRIDE    string `mapstructure:"TLS_OVERRIDE"`
```

Comment:
In keeping with the previous comment:
```suggestion
	TLS_HOSTNAME_OVERRIDE    string `mapstructure:"TLS_HOSTNAME_OVERRIDE"`
```